### DISCUSSION TITLE
Integration of clean energy forecast

### DISCUSSION THREAD

**Discussion by fennectech** [BOINC/boinc#5835] [Ideas]
**Describe the problem**
we have cleaner energy in certain parts of the day  would it be possible to use data like this to suspend when energy is less enviromentaly friendly

**Describe the solution you'd like**
id like to resume crunching when clean energy goes above a certain threshold and suspend when it drops below that

**Additional context**
Many paid APIs provide this data and perhaps one would be willing to donate their services to a program like boinc to help us improve our renewable utilization while minimizing use when renewable energy is less available thus reducing boinc's carbon impact. of course we would loose some compute power using this

**Comment by AenBleidd** [comment]
You can use this https://gist.github.com/CallMeFoxie/56e7ebd497ce5f979b801edb2a89f3a9 (or similar scripts) together with your SmartHome system via MQTT to control BOINC in a way you like.
If any SmartHome vendor is interested in integrating with BOINC - they can easily do that.
